src/App.js

```diff
@@ -4,7 +4,7 @@ import Detailed from './components/Detailed';
 import NotFound from './components/404';
 
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-import { DETAILED_PAGE } from './fileWithConstants';
+import { POKEMON } from './fileWithConstants';
```

Comment:
instead of `fileWithConstants` file use folder called `constants/routes` and then `constants/index` for more general constants